bibliography.bib

```diff
@@ -7142,3 +7142,474 @@ @Article {ProHsiKum2017IEEETR
+    doi          = {10.1007/BF00992698},
+    note         = {},
+    annote       = {},
+}
+
+@Article{MniKavSil-etal2015NATU,
+    title        = {Human-level control through deep reinforcement learning},
+    author       = Mnih_V # and # Kavukcuoglu_K # and # Silver_D # and # Rusu_A # and # Veness_J # and # Bellemare_MG # and # Graves_A # and # Riedmiller_M # and # Fidjeland_AK # and # Ostrovski_G,
+    journal      = NATU,
+    series       = {},
+    volume       = {518},
+    number       = {7540},
+    year         = {2015},
+    pages        = {529--533},
+    doi          = {10.1038/nature14236},
+    note         = {},
+    annote       = {},
+}
+
+@Article{Nob2006NATUBIOT,
+    title        = {What is a support vector machine?},
+    author       = Noble_WS,
+    journal      = NATUBIOT,
+    series       = {},
+    volume       = {24},
+    number       = {12},
+    year         = {2006},
+    pages        = {1565--1567},
+    doi          = {10.1038/nbt1206-1565},
+    note         = {},
+    annote       = {},
+}
+
+@Article{HorStiWhi1989NN,
+    title        = {Multilayer feedforward networks are universal approximators},
+    author       = Hornik_K # and # Stinchcombe_M # and # White_H,
+    journal      = NN,
+    series       = {},
+    volume       = {2},
+    number       = {5},
+    year         = {1989},
+    pages        = {359--366},
+    doi          = {10.1016/0893-6080(89)90020-8},
+    note         = {},
+    annote       = {},
+}
+
+@Article{BelHsuMaMan2019PNAS,
+    title        = {Reconciling modern machine-learning practice and the classical bias--variance trade-off},
+    author       = Belkin_M # and # Hsu_D # and # Ma_S # and # Mandal_S,
+    journal      = PNAS,
+    series       = {},
+    volume       = {116},
+    number       = {32},
+    year         = {2019},
+    pages        = {15849--15854},
+    doi          = {10.1073/pnas.1903070116},
+    note         = {},
+    annote       = {},
+}
+
+@Article{TriNol2011AL,
+    title        = {Engineering the evolution of self-organizing behaviors in swarm robotics: A case study},
+    author       = Trianni_V # and # Nolfi_S,
+    journal      = AL,
+    series       = {},
+    volume       = {17},
+    number       = {3},
+    year         = {2011},
+    pages        = {183--202},
+    doi          = {10.1162/artl_a_00031},
+    note         = {},
+    annote       = {},
+}
+
+@InCollection {NgCoaDie-etal2006iser,
+    title        = {Autonomous inverted helicopter flight via reinforcement learning},
+    author       = Ng_AY # and # Coates_A # and # Diel_M # and # Ganapathi_V # and # Schulte_J # and # Tse_B # and # Berger_E # and # Liang_E,
+    booktitle    = iser_2006,
+    editor       = Ang_MH # and # Khatib_O,
+    series       = {},
+    volume       = {21},
+    number       = {},
+    year         = {2006},
+    pages        = {363--372},
+    publisher    = pub_SP,
+    address      = Berlin_DEU # and # Heidelberg_DEU,
```

Comment:
Only include Berlin_DEU. This is our convention for dealing with the two addresses of Springer. Probably, we should also document that in the README.md file.